src/irmin-pack/unix/import.ml

```diff
@@ -0,0 +1,43 @@
+(*
+ * Copyright (c)2018-2021 Tarides <contact@tarides.com>
+ *
+ * Permission to use, copy, modify, and distribute this software for any
+ * purpose with or without fee is hereby granted, provided that the above
+ * copyright notice and this permission notice appear in all copies.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+ * WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ * MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+ * ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ * WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+ * ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+ * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+ *)
+
+include Irmin.Export_for_backends
+
+let src = Logs.Src.create "irmin.pack" ~doc:"irmin-pack backend"
+
+module Log = (val Logs.src_log src : Logs.LOG)
+
+module Int63 = struct
+  include Optint.Int63
+
+  let t = Irmin.Type.int63
+end
+
+type int63 = Int63.t [@@deriving irmin]
+
+let ( ++ ) = Int63.add
+let ( -- ) = Int63.sub
```

Comment:
```suggestion
include Irmin_pack.Import
```

Reply:
Is it fine to define twice `Logs.Src.create "irmin.pack" ~doc:"irmin-pack backend"`?

Reply:
I don't know, but it probably make sense to change the name. I've pushed a change